securedrop/tests/common.py

```diff
@@ -5,6 +5,8 @@
 
 import gnupg
 
+# Set environment variable so config.py uses a test environment
+os.environ['SECUREDROP_ENV'] = 'test'
```

Comment:
At some point, we might find another way to do this. We have this set in a few different places (also migrating this code above these module-level imports makes this file no longer PEP8 compliant - bad if we want to go in the direction of #886). Perhaps we could migrate some of the environmental variable setting necessary for our test running elsewhere (e.g. in `manage.py` or in the test runner (tox or pytest-env)). For now let's just leave as is and then come back after the hackathon.